sycl/include/sycl/ext/oneapi/bindless_images_descriptor.hpp

```diff
@@ -204,6 +205,11 @@ struct image_descriptor {
                               "or array_size must be 1.");
       }
       return;
+    case image_type::gather:
+      if (this->width == 0 || this->height == 0 || this->depth != 0) {
+        throw sycl::exception(sycl::errc::invalid, "Gather images must be 2D.");
```

Comment:
This error code should be mentioned in the extension document, I can't find it mentioned right now. Though to be fair, I don't think other types have this documented, so we can leave it for a future patch.

Reply:
OK, yeah I can add it for completeness at  least for this gather documentation.

Reply:
Done